index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Apple</title>
+    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+</head>
+<body>
+    <main>
+        <nav>
+            <ul>
+                <li id="apple-icon"><a href=#><i class="fab fa-apple"></i></a></li>
+                <li><a href=#>Store</a></li>
+                <li><a href=#>Mac</a></li>
+                <li><a href=#>iPod</a></li>
+                <li><a href=#>iPhone</a></li>
+                <li><a href=#>iPad</a></li>
+                <li><a href=#>iTunes</a></li>
+                <li><a href=#>Support</a></li>
+                <li><i class="fa fa-search"></i><input type="text"></li>
+            </ul>
+        </nav>
+        <div class="title"></div>
```

Comment:
These divs are also links too, just correct that, merge to master and Submit the project completion form, Good Job.